lib/messages.js

```diff
@@ -827,6 +829,110 @@ util.inherits(GetHeaders, GetBlocks);
 module.exports.GetHeaders = Message.COMMANDS.getheaders = GetHeaders;
 
 /**
+ * Request peer to apply a bloom filter to 'inv' messages sent back
+ *
+ * @name P2P.Message.filterload
+ * @param{BloomFilter} filter - a BloomFilter object
+ */
+function FilterLoad(filter) {
+  this.command = 'filterload';
+  $.checkArgument(_.isUndefined(filter) || filter instanceof BloomFilter,
+    'Object or undefined required for filter object');
+
+  // TODO: Are these defaults (1,0.01) sane?
+  this.filter = filter || BloomFilter.create(1, 0.01);
```

Comment:
I think we should either provide a way to configure these options for the bloom filter, or require a prior instance.